smithy-event-stream-rpc-cpp/src/main/resources/software/amazon/smithy/eventstreamrpc/cpp/model/macros/StructUnion.h.ftl

```diff
@@ -29,31 +32,30 @@
     <#local PascalCase = fn_camel_to_pascal.apply(memberShape.getMemberName())>
     <#local camelCase = memberShape.getMemberName()>
     <#local memberTargetShape = context.getShape(memberShape.getTarget())>
+    <#local memberDocMessage = "">
+    <#if memberShape.hasTrait("documentation")>
```

Comment:
Any reason why we "remove" the `deprecationMessage`?

Reply:
Currently, the only way to find out that method is deprecated is to look at it in the header file (to be fair, clion and probably other IDEs show this message as part of the method docs). I wanted to make it visible in the API docs as well.